aleph/views/serializers.py

```diff
@@ -118,19 +118,20 @@ def _serialize(self, obj):
         obj["shallow"] = obj.get("shallow", True)
         if self.nested or not obj["writeable"]:
             obj.pop("has_password", None)
+            obj.pop("has_api_key", None)
+            obj.pop("api_key_expires_at", None)
             obj.pop("is_admin", None)
             obj.pop("is_muted", None)
             obj.pop("is_tester", None)
             obj.pop("is_blocked", None)
-            obj.pop("api_key", None)
             obj.pop("email", None)
             obj.pop("locale", None)
             obj.pop("created_at", None)
             obj.pop("updated_at", None)
         if obj["type"] != Role.USER:
```

Comment:
Also pop `has_api_key` and `api_key_expires_at` in this case. Probably that’s not strictly necessary, but it’s better.